app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -29,15 +31,9 @@ def access_token(input)
             conjur_account: input.account
         )
 
-        # Prepare ID token introspect request
-
-        # send id token to OIDC Provider
-
-        # Get JSON from OIDC Provider
-
-        # Validate ID Token is active
+        id_token_attributes = @decode_and_verify_id_token.(oidc_client_configuration.provider_uri, request_body.id_token)
```

Comment:
Can you please split this up over a few lines so it isn't so long (same with the line below)?